executor/show_test.go

```diff
@@ -310,12 +310,9 @@ func (s *testSuite) TestShow(c *C) {
 	))
 
 	tk.MustExec(`drop table if exists t`)
-	tk.MustExec(`create table t (a int) default charset=abcdefg`)
-	tk.MustQuery(`show create table t`).Check(testutil.RowsWithSep("|",
-		"t CREATE TABLE `t` (\n"+
-			"  `a` int(11) DEFAULT NULL\n"+
-			") ENGINE=InnoDB DEFAULT CHARSET=abcdefg",
-	))
+
+	_, err := tk.Exec("create table t (a int) default charset=abcdefg")
```

Comment:
I think this test case should move away from show_test.go . Maybe ddl_db_test.go is a good place for it.

Reply:
Yes, I think so.